### ISSUE TITLE
Fails HTML Validation

### ISSUE THREAD

**Issue by shoesforindustry** [distantnative/kirby-oembed#21]
Trying this with Vimeo, I'm getting invalid HTML errors, any idea how to fix?

Validation Output: 7 Errors

+ Error Line 23, Column 4089: & did not start a character reference. (& probably should have been escaped as &amp;.)
    `…r.vimeo.com/video/127047977?autoplay=1&byline=0&portrait=0" frameborder="0" we…`

+ Error Line 23, Column 4098: & did not start a character reference. (& probably should have been escaped as &amp;.)
    `…om/video/127047977?autoplay=1&byline=0&portrait=0" frameborder="0" webkitAllow…`

+ Error Line 23, Column 4183: Attribute webkitallowfullscreen not allowed on element iframe at this point.

   ` …een mozallowfullscreen allowFullScreen></iframe></div> </figure><h3>Heading</h…`

    Attributes for element iframe:
        Global attributes
        src
        srcdoc
        name
        sandbox
        seamless
        allowfullscreen
        width
        height

+ Error Line 23, Column 4183: Attribute mozallowfullscreen not allowed on element iframe at this point.

   ` …een mozallowfullscreen allowFullScreen></iframe></div> </figure><h3>Heading</h…`

    Attributes for element iframe:
        Global attributes
        src
        srcdoc
        name
        sandbox
        seamless
        allowfullscreen
        width
        height

+ Error Line 23, Column 4183: The frameborder attribute on the iframe element is obsolete. Use CSS instead.

   ` …een mozallowfullscreen allowFullScreen></iframe></div> </figure><h3>Heading</h…`

+ Error Line 23, Column 3904: An img element must have an alt attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.

    `…try.net/assets/oembed/oembed-play.png"></div><img src="http://test.shoesforind…`

+ Error Line 23, Column 4025: An img element must have an alt attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.

    `…c0fdca30f9172e7c7a5.jpg" class="thumb"><iframe src="http://player.vimeo.com/vi…`

**Comment by distantnative** [comment]
Some of the errors I can't fix since it's simply the embed code provided by the platforms themselves (like Vimeo). I still tried to improve validation by removing attributes such as `mozallowfullscreen`.

I also added an alt attribute to the play-icon image tag, so that the pre-last error should be resolved.

To the last error: It seems like you are using an older version of the plugin as the current one doesn't use an image tag for the thumb anymore, but embeds it as a background image. So that should be solved as well.

Let me know how it works for yo!

**Comment by shoesforindustry** [comment]
Thanks Nico, I upgraded to 0.7, I don't use JS for lazyloading, all my code the same as before.
Not output doesn't work, note '&lt;i' etc. around ifame.

```html
<div class="oembed oembed-video" style="padding-top:56.25%">
  <div class="play">
    <img src="http://bigmac.local:5757/assets/oembed/oembed-play.png" alt="Play">
  </div>
  <div class="thumb" style="background-image: url(http://i.vimeocdn.com/video/517696084_1280.jpg)">
    
  </div>&lt;iframe src="http://player.vimeo.com/video/127047977?autoplay=1&amp;byline=0&amp;portrait=0"    allowFullScreen&gt;&lt;/iframe&gt;
</div> 
```

My Code:
```php
          <?php echo $page->video()->oembed(); ?>
```

**Comment by distantnative** [comment]
arg stupid me, sorry. will fix this in an 0.7.1 asap

p.s.: btw your code did change around the other things, e.g. with `alt="Play"` or the thumb as background-image.
